packages/coordinator/src/components/CoordinatorScreen/CoordinatorScreen.tsx

```diff
@@ -13,11 +14,13 @@ export type CoordinatorScreenProps = {
   className?: string;
   addAppointmentFabProps?: {
     hospital: Hospital;
-    timestamp?: number;
   };
 };
 
 export default function CoordinatorScreen(props: CoordinatorScreenProps) {
+  const { timestamp } = useParams<{ timestamp: string }>();
+  const currentScheduleTime = timestamp ? parseInt(timestamp) : undefined;
```

Comment:
This will affect all screens that have this button, and I don't think it's a good idea.
I believe the logic in the Schedule screen should remain the same, while in the DonationDay screen should change.

Reply:
Got you, my bad 
fixed